boot/runtimes/psyntax-mosh/psyntax/library-manager.ss

```diff
@@ -131,7 +131,7 @@
 
   (define library-extensions
     (make-parameter
-      '(".mosh.sls" ".sls" ".ss" ".scm")
+      '(".sld" ".sls" ".ss" ".scm")

```

Comment:
(It seems we needed to remove `.mosh.sls` to avoid `.mosh.mosh.sls`. `.mosh` addition is handled in `main.ss`)

Reply:
> (It seems we needed to remove `.mosh.sls` to avoid `.mosh.mosh.sls`. `.mosh` addition is handled in `main.ss`)

Oh. I had no idea. thanks.